defaults-jalien.sh

```diff
@@ -0,0 +1,12 @@
+package: defaults-jalien
+version: v1
+overrides:
+  xrootd:
```

Comment:
Down below you are also setting `tag:` in the `xrootd.sh` recipe, just keep those overrides here and drop the ones in `xrootd.sh`.

Reply:
But the tag setting in xrootd is needed for production isn't it?

Reply:
So, the one in `xrootd.sh` should not be changed. That one is used for production. You should only override `tag:`, `version:` and `source:` in `defaults-jalien.sh`, that was my point.

Reply:
I thought we agreed to update to the official repo, and not
use our own fork? Unless you wish to do the xrootd recipe
update in a different PR?

Reply:
One point however is that @costing requires the xrootd bin and lib dirs
set up at runtime, and I believe that this can only be done by adding
a modulefile to the current xrootd recipe, which has none.

Reply:
So what I am saying is:

* we don't need to set `tag:`, `version:`, `source:` _twice_, we need it once...
* ...and this one place should be _here_, in the `defaults-jalien.sh` configuration
* adding a modulefile to `xrootd.sh` is very fine, I haven't made any "negative" comment about it 🙂